package.nls.json

```diff
@@ -15,7 +15,7 @@
     "staticWebApps.deleteConfirmation.ClickButton": "Prompts with a warning dialog where you click a button to delete.",
     "staticWebApps.createHttpFunction": "Create HTTP Function...",
     "staticWebApps.browse": "Browse Site",
-    "staticWebApps.showActions": "Show Actions",
+    "staticWebApps.showActions": "Show in GitHub",
```

Comment:
What about "Open in GitHub" to match "Open in Portal"?

Reply:
@fiveisprime Didn't like that for some reason 🤔

Reply:
"Open in GitHub" is fine 😅